org.mixedrealitytoolkit.uxcore/Slider/SliderSounds.cs

```diff
@@ -109,6 +109,13 @@ public class SliderSounds : MonoBehaviour
         /// </summary>
         public float MinSecondsBetweenTicks { get => minSecondsBetweenTicks; set => minSecondsBetweenTicks = value; }
 
+        [SerializeField]
+        [Tooltip("The source of the grab and release sounds")]
+        private AudioSource grabReleaseAudioSource = null;
```

Comment:
Since you're making this serializable fields.   Do you see benefit in exposing these as properties, so to enable component creation at runtime?

Reply:
I can see doing that. Will update the PR shortly.

Reply:
Public accessors added.